src/main/scala/io/github/paoloboni/binance/spot/parameters/v3/KLines.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2022 Paolo Boni
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package io.github.paoloboni.binance.spot.parameters.v3
+
+import io.github.paoloboni.binance.common.Interval
+
+import java.time.Instant
+
+final case class KLines(
+    symbol: String,
+    interval: Interval,
+    startTime: Option[Instant],
+    endTime: Option[Instant],
```

Comment:
Model supporting optional startTime/endTime. This model is used by `SpotRestApiV3Impl#getKlines`. The existing `SpotApi#getKlines` is marked as deprecated, along with all rest operations defined under `SpotApi`